tools/test/test_osbuild_image_info.py

```diff
@@ -135,3 +138,34 @@
 def test_read_boot_entries(tmp_path, fake_tree, entries):
     make_fake_tree(tmp_path, fake_tree)
     assert osbuild_image_info.read_boot_entries(tmp_path / "boot") == entries
+
+
+def make_fake_iso(iso_tree, output_dir) -> str:
+    iso_path = os.path.join(output_dir, "image.iso")
+    subprocess.run(["genisoimage", "-o", iso_path, "-R", "-J", iso_tree], check=True)
+    return iso_path
+
+
+def test_analyse_iso_fail(tmp_path):
```

Comment:
This needs root to run the mount, right? do we need to have a skipif not root here?

Reply:
Yes, I'm already on it...